how-to/rpi-camera.rst

```diff
@@ -112,9 +112,10 @@ rpicam-apps
 ===========
 
 .. important::
-    While the AI Camera module is supported in Ubuntu, the post-processing stages require the imx500-firmware
+    While the AI Camera module is supported in Ubuntu, the post-processing stages require the ``imx500-firmware``
     package, which can't be included in Ubuntu due to licensing issues. So anything in the `AI camera documentation`_ might
-    not work out of the box. We are working on fixing the licensing issues with the involved parties.
+    not work out of the box. We are working on fixing the licensing issues with the involved parties. Till then, you can
```

Comment:
We should [avoid contractions](https://docs.ubuntu.com/styleguide/en#contractions) like "Till"